### PR TITLE
creating burger menu in header + styled it

### PR DESCRIPTION
**Pull request by camillaume**

### CODE REVIEW

**Review by n-faycel — APPROVED** [review]
ok

**Review by n-faycel — DISMISSED** [review]

**Review by LuigiMenez — APPROVED** [review]
No reading of the sm

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
no reading of the sm

**Review by n-faycel — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Beware of the empty line at the beginning of the file

File: frontend/src/components/Archi/Header/index.jsx [hunk 1]
```diff
@@ -1,12 +1,24 @@
+
```
Comment (LoicBrassart) [inline frontend/src/components/Archi/Header/index.jsx:1]: ?

**Review by LoicBrassart — APPROVED** [review]
Saw a (minor) Evil, bu Luigi is way too nice with you >_>

**Pull request merged**